src/containers/Main/style.css

```diff
@@ -2,4 +2,10 @@
     width: 100%;
```

Comment:
non footer related changes would be better in another branch

Reply:
I thought I had done this in another branch already - do you need me to redo this?